### PR TITLE
Subscription endpoint changes

### PR DESCRIPTION
**Pull request by Christoffer-Cortes**
### Applicable Issues
See Eiffel Intelligence back-end issue:
https://github.com/eiffel-community/eiffel-intelligence/issues/343

### Description of the Change
Endpoints are now changed according to the issue.
I have tried manual CURL.

### Sign-off
Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Christoffer Cortes Sjöwall, christoffer.cortes.sjowall@ericsson.com

### CODE REVIEW

File: src/main/resources/static/js/common.js [hunk 1]
```diff
@@ -322,8 +322,7 @@ function executeIfLdapIsDeactivated() {
 }
 
 function downloadSubscriptions(subscriptionData, subscriptionNames) {
-    var foundSubscriptions = subscriptionData.foundSubscriptions;
```
Comment (a-breid-ericsson) [inline src/main/resources/static/js/common.js]: Ensure that the back end does not return the json structure that was before {"notFoundSubscriptions":[], "foundSubscriptions":[]} But should now be ["subscription1", "subscription2",...]

File: src/main/resources/static/js/common.js [hunk 2]
```diff
@@ -322,8 +322,7 @@ function executeIfLdapIsDeactivated() {
...
-    var jsonData = JSON.stringify(foundSubscriptions, null, 2);
+    var jsonData = JSON.stringify(subscriptionData, null, 2);
```
[the remark this answers is not in the document]
Comment (a-breid-ericsson) [reply src/main/resources/static/js/common.js]: Note to Christoffer: null is a "replacer" not sure what it is, 2 is indention.

File: src/main/resources/static/js/common.js (hunk 1 above)
Comment (Christoffer-Cortes) [reply src/main/resources/static/js/common.js]: Tested it manually and it works.

File: src/main/resources/static/js/common.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Christoffer-Cortes) [reply src/main/resources/static/js/common.js]: I've put them in variables now.

File: wiki/markdown/curl-examples.md [hunk 3]
```diff
@@ -321,7 +321,16 @@ Example curl commands to these endpoints [can be found here](https://github.com/
     <tr>
         <td>/subscriptions</td>
         <td>GET</td>
-        <td>Fetches all available subscriptions from the back-end</td>
+        <td>
+            Fetches all available subscriptions from the back-end.
+            Selected subscriptions can be fetched by using the optional subscriptionNames parameter
+            and a comma separated list e.g. subscriptionNames={name1},{name2}
```
Comment (e-pettersson-ericsson) [inline wiki/markdown/curl-examples.md]: Perhaps leave out the curly braces in the example to be more clear? How about "?subscriptionNames=MyFirstSubscription,MySecondSubscription"

File: src/main/resources/static/js/common.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tobiasake) [reply src/main/resources/static/js/common.js]: ok

**Pull request merged**